src/main/java/DukeDate.java

```diff
@@ -0,0 +1,23 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class DukeDate {
+    private LocalDate localDate;
+
+    public DukeDate(String datetimeString) throws DukeException {
+        String[] datetimeArray = datetimeString.split(" ");

```

Comment:
I'm not sure about this but should the variable names be dateTimeArray and dateTimeString instead?